lib/types/types.ex

```diff
@@ -44,8 +44,4 @@ defmodule Types do
   @type withdrawal_index :: uint64
   @type payload_id :: bytes8
   @type blob_index :: uint64
-  # Max size: BYTES_PER_FIELD_ELEMENT * FIELD_ELEMENTS_PER_BLOB
-  @type blob :: binary
```

Comment:
Let's keep these in here, to keep changes to a minimum.

Reply:
So i will be importing only the blob type from types.ex.